File: 1074.py
loop = int(input())

for i in range(loop):
    n = int(input())
    if n == 0:
        print("NULL")
    elif n % 2 == 0 and n > 0:
        print("EVEN POSITIVE")
    elif n % 2 == 0 and n < 0:
        print("EVEN NEGATIVE")
    elif n % 2 == 1 and n > 0:
        print("ODD POSITIVE")
    elif n % 2 == 1 and n < 0:
        print("ODD NEGATIVE")

File: Dart/1074.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync());
  for (int i = 0; i < n; i++) {
    int x = int.parse(stdin.readLineSync());
    if (x == 0) {
      print("NULL");
    } else if ((x % 2 == 0) && (x > 0)) {
      print("EVEN POSITIVE");
    } else if ((x % 2 == 0) && (x < 0)) {
      print("EVEN NEGATIVE");
    } else if ((x % 2 != 0) && (x > 0)) {
      print("ODD POSITIVE");
    } else {
      print("ODD NEGATIVE");
    }
  }
}
